apps/nextjs-website/src/lib/guideListPages.ts

```diff
@@ -0,0 +1,49 @@
+import { GuidesPageProps } from '@/app/[productSlug]/guides/page';
+import { StrapiGuideListPages } from './strapi/guideListCodec';
+import { mergeProductWithStaticContent } from './products';
+import { GuidesSectionProps } from '@/components/molecules/GuidesSection/GuidesSection';
+
+type StaticGuideListPages = readonly GuidesPageProps[];
+
+export function makeGuideListPagesProps(
+  strapiGuideListPages: StrapiGuideListPages,
+  staticGuideListPages: StaticGuideListPages
+): readonly GuidesPageProps[] {
+  return [
+    ...strapiGuideListPages.data.map(({ attributes }) => {
+      const product = mergeProductWithStaticContent(
+        attributes.product.data.attributes
+      );
+      const guidesSections: readonly GuidesSectionProps[] = [
+        ...attributes.guidesByCategory.map(({ category, guides }) => ({
+          title: category,
+          guides: guides.data.map(({ attributes }) => ({
+            title: attributes.title,
+            description: {
+              title: 'guideListPage.cardSection.listItemsTitle', // this is translations path and it will be tranlated by the component
```

Comment:
```suggestion
              title: 'guideListPage.cardSection.listItemsTitle', // this is translations path and it will be translated by the component
```